torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -238,6 +291,57 @@ Operation BroadOp(const Node* node) {
   };
 }
 
+const RegisterOperators MKLDNNHardSwishOpReg({
+    torch::jit::Operator(
+        "prim::MKLDNNHardSwish_(Tensor(a!) self) -> Tensor(a!)",
+        createUnaryOp(
+            [](at::Tensor output, at::Tensor input) {
+              at::hardswish_out(output, input);
```

Comment:
For these operators, we could just call `at::cpu::hardswish_out`. Since we are already creating a custom operator we might as well avoid the overhead